regulations/static/regulations/js/source/models/comment-model.js

```diff
@@ -3,12 +3,39 @@
 var $ = require('jquery');
 var Backbone = require('backbone');
 
-module.exports = Backbone.Model.extend({
+var comment_model = Backbone.Model.extend({
   defaults: {
     docId: '',
     label: '',
     comment: '',
     commentHtml: '',
-    files: []
+    files: [],
+    indexes: []
   }
 });
+
+var index_comparator = function(first, second) {
```

Comment:
Minor, but we use camelCase for other JS stuff; I'm surprised the linter didn't complain